infrastructure/variables.tf

```diff
@@ -15,3 +19,29 @@ variable "deployment_namespace" {}
 variable "common_tags" {
   type = "map"
```

Comment:
Specify the type as `map(string)`.  This should resolve a warning that will be displayed in the log when the infrastructure part of the pipeline runs.

Reply:
Type has been changed as requested.